README.md

```diff
@@ -1,25 +1,50 @@
 # Substrate Front End Template
 
-## Purpose
+This template allows you to create a front-end application that connects to a
+[Substrate](https://github.com/paritytech/substrate) node back-end with minimal
+configuration. To learn about Substrate itself, visit the
+[Substrate Developer Hub](https://substrate.dev).
 
-The purpose of this front-end template is to allow you to quickly create a
-front-end application that connects to a [Substrate](https://github.com/paritytech/substrate)
-node without much configuration and simplify the underlying connection and
-handling of [Polkadot-JS](https://polkadot.js.org/api/).
+The template is built with [Create React App](https://github.com/facebook/create-react-app)
+and [Polkadot js API](https://polkadot.js.org/api/). Familiarity with these tools
+will be helpful, but the template strives to be self-explanatory. To learn how
+this template was built, visit the
+[Substrate Front-End Tutorial](https://substrate.dev/docs/en/tutorials/substrate-front-end/).
 
-The encapsulation is handled by components inside `substrate-lib`. Substrate
-connection information is exposed out via `substrate-lib/useSubstrate.js`
-[custom hook](https://reactjs.org/docs/hooks-custom.html).
+## Using The Template
+
+### Installation
+
+The code can be installed using [git](https://git-scm.com/) and [yarn](https://yarnpkg.com/).
+```bash
+# Clone the repository
+git clone https://substrate-developer-hub/substrate-front-end-template.git
+cd ./substrate-front-end-template
+```
+
+## Usage
+You can start the template in development mode to connect to a locally running node
+```bash
+yarn start
+```
+
+You can also start it in production mode...
+TODO @jimmychu What are the details about this again?
```

Comment:
`yarn build` triggers the production settings to kick in.

Reply:
Added those details. I'm open to suggestions on how to format that part.